src/qibolab/oneQ_emulator.py

```diff
@@ -0,0 +1,56 @@
+from qibolab.emulator import create_runcard_emulator
+from qibolab.instruments.simulator.models import general_no_coupler_model
+from qibolab.instruments.simulator.pulse_simulator import (
+    PulseSimulator,
+    get_default_simulation_config,
+)
+
+# device parameters: frequency in Hz, time in s
+device_name = "ibmfakebelemQ0"
+sampling_rate = 4500000000.0
+readout_error = [0.01, 0.02]
+lo_freq = 5090167234.445013
+anharmonicity = -336123005.1821652
+drive_freq = 5090167234.445013
+rabi_freq = 125457538.19061986
+T1 = 8.857848970762537e-05
+T2 = 0.00010679794866226273
+nlevel = 3
+
+# simulation parameters
+sim_sampling_boost = 10
+simulate_dissipation = True
+instant_measurement = True
+
+
+def create_oneQ_emulator(runcard_folder: str):
+    """Create a one qubit emulator platform."""
+
+    model_params_dict = {
+        "device_name": device_name,
+        "topology": [],
+        "nqubits": 1,
+        "ncouplers": 0,
+        "qubits_list": ["0"],
+        "couplers_list": [],
+        "sampling_rate": sampling_rate,
+        "readout_error": {0: readout_error},
+        "drive_freq": {"0": drive_freq},
+        "T1": {"0": T1},
+        "T2": {"0": T2},
+        "lo_freq": {"0": lo_freq},
+        "rabi_freq": {"0": rabi_freq},
+        "anharmonicity": {"0": anharmonicity},
+        "coupling_strength": {},
+    }
+
+    model_config = general_no_coupler_model.generate_model_config(
+        model_params_dict, nlevels_q=[nlevel]
+    )
+    simulation_config = get_default_simulation_config(sim_sampling_boost)
+    simulation_config.update({"simulate_dissipation": simulate_dissipation})
+    simulation_config.update({"instant_measurement": instant_measurement})
+
+    pulse_simulator = PulseSimulator(simulation_config, model_config)
+
+    return create_runcard_emulator(runcard_folder, pulse_simulator)
```

Comment:
I believe this function could be inlined here, since even the channel definition is platform dependent, while Qibolab is the library, so it should not commit to any specific platform.

Reply:
Sorry I do not understand your comment here, could you please elaborate?

Reply:
You could place the content of the function here, instead of the function call (i.e. inlining).

Reply:
I see, to confirm, do you mean that I should not import and use `create_runcard_emulator` here but have it explicitly constructed here?

Reply:
Yes, exactly.

The rationale is that the content of `create_runcard_emulator()` is quite specific to this platform, and (essentially because of this) it resembles the content of the `platform.py` files that we have in `qibolab_platforms_qrc`, e.g..

It is definitely correct to attempt factorizing common patterns, but in this case we are already trying to do it in the general Qibolab, and you have a single instance of platform creation (in Python), thus you are deduplicating a single occurrence. Modularizing it could be good in any case, because of readability and isolation, but factorizing early often means redoing it later on, together with some reshuffling (because for as long as there is only one example, it is not necessarily clear what is going to be repeated).

Reply:
Fixed in Fixed in 76d5cb5 and 15307d6